src/visualizers/widgets/TextEditor/TextEditorWidget.js

```diff
@@ -1,79 +1,111 @@
-/*globals $, define*/
+/*globals $, define, monaco */
 /*jshint browser: true*/
 
 define([
-    'ace/ace',
     'underscore',
-    './completer',
     'js/Utils/ComponentSettings',
+    'text!./MonacoLanguages.json',
+    'vs/editor/editor.main',
     'jquery-contextMenu',
     'css!./styles/TextEditorWidget.css'
 ], function (
-    ace,
     _,
-    Completer,
-    ComponentSettings
+    ComponentSettings,
+    MonacoLanguages
 ) {
     'use strict';
 
-    var TextEditorWidget,
-        WIDGET_CLASS = 'text-editor',
+    const WIDGET_CLASS = 'text-editor',
         LINE_COMMENT = {
             python: '#',
             lua: '--',
             yaml: '#'
         };
 
-    TextEditorWidget = function (logger, container, config={}) {
-        this.logger = logger.fork('Widget');
+    MonacoLanguages = JSON.parse(MonacoLanguages);
+    const DEFAULT_THEMES = ['vs-dark', 'vs', 'hc-black'];
```

Comment:
It might be nice to have a couple more themes available. Maybe 5 or 6?

Reply:
https://github.com/brijeshb42/monaco-themes. Similar to https://github.com/deepforge-dev/deepforge/pull/1800#discussion_r459724791. Which would be the best course?